src/plugins/discover/public/application/main/utils/fetch_documents.ts

```diff
@@ -47,7 +36,7 @@ export const fetchDocuments = (
     name: 'discover',
     description: 'fetch documents',
     url: window.location.pathname,
-    id: '',
```

Comment:
testet by the instructions of #108016, works:
<img width="1085" alt="Bildschirmfoto 2021-12-17 um 09 57 03" src="https://user-images.githubusercontent.com/463851/146517642-aae0c553-5ede-4b34-99f9-25bd651ff5a9.png">